static/script.js

```diff
@@ -18,20 +17,43 @@
   const AGENCY_KEY = "AGBUR";
   const MIN_ZOOM = 6;
 
-  L.tileLayer(
+  // create the tile layer with correct attribution
+
+  var hotLayer = new L.tileLayer('http://{s}.tile.openstreetmap.fr/hot/{z}/{x}/{y}.png', 
+  	{
+  		maxZoom: 20, 
+  		attribution: 'Data \u00a9 <a href="http://www.openstreetmap.org/copyright"> OpenStreetMap Contributors </a> Tiles \u00a9 HOT'
+  	}),
+  darkmatterLayer = L.tileLayer(
     "https://cartodb-basemaps-{s}.global.ssl.fastly.net/dark_nolabels/{z}/{x}/{y}.png",
     {
       maxZoom: 18,
       attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>, &copy; <a href="https://carto.com/attribution">CARTO</a>'
-    }
-  ).addTo(myMap);
-
-  const labelLayer = L.tileLayer(
-    "https://cartodb-basemaps-{s}.global.ssl.fastly.net/dark_only_labels/{z}/{x}/{y}.png",
+    });
+  var thunderforest = L.tileLayer(
+    "https://a.tile.thunderforest.com/landscape/{z}/{x}/{y}.png",
```

Comment:
Also thunderforest's terms say that an api key is required for all services (see http://thunderforest.com/terms/ and http://thunderforest.com/docs/apikeys/) so it should be specified as part of the tile layer url. Having to register to use _might_ qualify this as a SaaS, which means it probably needs to go through our SaaS review process :\